README.md

```diff
@@ -1,7 +1,6 @@
 # Unattended Upgrades module for Puppet
 
 [![Build Status](https://travis-ci.org/voxpupuli/puppet-unattended_upgrades.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-unattended_upgrades)
-[![Code Coverage](https://coveralls.io/repos/github/voxpupuli/puppet-unattended_upgrades/badge.svg?branch=master)](https://coveralls.io/github/voxpupuli/puppet-unattended_upgrades)
```

Comment:
This project doesn't have a lib dir.  We no longer run coveralls on it.  In the distant past, we accidentally also included coverage of things in fixtures.  This is where the 76% coverage result came from. https://coveralls.io/builds/8635772

Reply:
a bit sad that we drop that, snice 76% is quite good :D